proto/info/info.proto

```diff
@@ -930,8 +930,7 @@ message ZInfoApp {
   repeated ZInfoNetwork network = 16;       // up/down; allocated IP
   repeated string volumeRefs = 17;          // volume UUIDs
   repeated ZInfoSnapshot snapshots = 18;    // optional field, used to send list of snapshots on device
-    // optional; information about usage of every patchEnvelope assigned to app
-  repeated ZInfoPatchEnvelopeUsage patchEnvelope = 19; 
+  // deprecated = 19; struct was removed
```

Comment:
How about using
 reserved: 19; // deprecated

Reply:
Will update this and yetus complaints

Reply:
@eriknordmark do you know if field number 10 in this struct is used? I'd also reserved it

Reply:
I reserved field number 10 just in case